src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpRequestStream.cs

```diff
@@ -132,7 +132,7 @@ public override Task WriteAsync(byte[] buffer, int offset, int count, Cancellati
 
         public override void Write(byte[] buffer, int offset, int count)
         {
-            WriteAsync(buffer, offset, count, CancellationToken.None).GetAwaiter().GetResult();
+            WriteAsync(buffer, offset, count, CancellationToken.None).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
This one isn't necessary and will be a nop.  It's only necessary when await'ing, not when synchronously blocking.